Kconfig

```diff
@@ -1295,6 +1295,30 @@ menu "LVGL configuration"
 		config LV_USE_TFT_ESPI
 			bool "Use TFT_eSPI driver"
 			default n
+
+		config LV_USE_EVDEV
+			bool "Use evdev input driver"
+			default n
+		config LV_EVDEV_SWAP_AXES
+			bool "Swap horizontal and vertical axes for evdev pointer events"
+			depends on LV_USE_EVDEV
+			default n
+		config LV_EVDEV_CALIBRATE
+			bool "Transform coordinates of evdev pointer events"
+			depends on LV_USE_EVDEV
+			default n
+		config LV_EVDEV_HORIZONTAL_MIN
+			int "Minimum horizontal coordinate to transform evdev pointer events"
+			depends on LV_EVDEV_CALIBRATE
+		config LV_EVDEV_HORIZONTAL_MAX
+			int "Maximum horizontal coordinate to transform evdev pointer events"
+			depends on LV_EVDEV_CALIBRATE
+		config LV_EVDEV_VERTICAL_MIN
+			int "Minimum vertical coordinate to transform evdev pointer events"
+			depends on LV_EVDEV_CALIBRATE
+		config LV_EVDEV_VERTICAL_MAX
```

Comment:
In v9 LVGL's multi display feature should be better supported. For this these should be runtime configs. E.g. `lv_evdev_set_calibration()`